pkg/operation/seed/scheduler/gardener_kube_scheduler.go

```diff
@@ -65,6 +66,7 @@ func New(
 	client client.Client,
 	namespace string,
 	image string,
+	leaseResourceName string,
```

Comment:
Is it really necessary that somebody has to specify this here? Can't it be "hard-coded" by the package? What's the use-case?

Reply:
Reworked the PR.